src/components/common/Layout/components/Background.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, BoxProps } from "@chakra-ui/react"
+import { useThemeContext } from "components/[guild]/ThemeContext"
+import Image from "next/image"
+
+type Props = {
+  image?: JSX.Element
```

Comment:
We should remove the image and imageUrl props, remove the background prefix from the other props, and handle that we either have an image or a background prop. If we have the latter, we should just add `& BoxProps` instead of the backgroundProps we have now, so they can be spread to the element